core/controllers/skill_editor_test.py

```diff
@@ -267,3 +269,33 @@ def test_skill_publish_handler_fails(self):
                 self.url, {'version': 1}, csrf_token=csrf_token,
                 expected_status_int=401)
         self.logout()
+
+
+class SkillEditorQuestionHandlerTests(BaseSkillEditorControllerTests):
+
+    def test_skill_editor_question_handler_updates_question_summary_dicts(self):
+        self.login(self.ADMIN_EMAIL)
+
+        response = self.get_json(
+            '%s/%s' % (feconf.SKILL_EDITOR_QUESTION_URL, self.skill_id))
+        question_summary_dicts = response['question_summary_dicts']
+
+        self.assertEqual(question_summary_dicts, [])
+
+        question_id = question_services.get_new_question_id()
+        self.save_new_question(
+            question_id, self.admin_id,
+            self._create_valid_question_data('ABC'))
+        question_services.create_new_question_skill_link(
+            question_id, self.skill_id, constants.DEFAULT_SKILL_DIFFICULTY)
+
+        response = self.get_json(
+            '%s/%s' % (feconf.SKILL_EDITOR_QUESTION_URL, self.skill_id))
+        question_summary_dicts = response['question_summary_dicts'][0]
```

Comment:
nit: this is a single dict, so please name the variable `question_summary_dict` instead.

Reply:
Still pending.

Reply:
Done